### PR TITLE
feat(query): attach short sql in context

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

## Changelog

 
- Improvement
 

## Related Issues

Fixes #issue

### CODE REVIEW

File: query/src/sql/sql_common.rs [hunk 1]
```diff
@@ -85,4 +85,12 @@ impl SQLCommon {
             ))),
         }
     }
+
+    pub fn short_sql(query: &str) -> String {
+        if query.len() >= 64 && query[..=6].eq_ignore_ascii_case("INSERT") {
```
Comment (zhang2014) [inline query/src/sql/sql_common.rs:90]: Maybe need trim for `.    INSERT INTO table()`

Comment (sundy-li) [reply query/src/sql/sql_common.rs:90]: I don't want to use parser/regexp here, so this it's the simple way to store short insert sqls.

Comment (zhang2014) [reply query/src/sql/sql_common.rs:90]: Okay.

**Comment by zhang2014** [comment]
`mysql: Failed to download resource "protobuf_bottle_manifest"` restarted failure test

**Comment by BohuTANG** [comment]
> `mysql: Failed to download resource "protobuf_bottle_manifest"` restarted failure test

Looks not related to this PR :/

**Pull request merged**